### DISCUSSION TITLE
po-dynamic-form/p-load - Não muda a propriedade

### DISCUSSION THREAD

**Discussion by leandrodefaria** [po-ui/po-angular#1154] [Tire suas dúvidas]
Inclui um p-load no dynamic-form, porém o mesmo não está mudando a propriedade de um componente para visible false. É necessário mais alguma parametrização?

>   <po-dynamic-form
    [p-fields]="fields"
    [p-load]="getUserDocument.bind(this)"
    [p-value]="valueFields">
  </po-dynamic-form>

>  getUserDocument(value: any): PoDynamicFormLoad {

    const cpfField = { property: 'cpf', visible: false };

    return {
      fields: [cpfField]
    };
  }`

**Comment by alinelariguet** [comment]
Ocorre algum erro no console? temos um sample no nosso portal na documentação do po-dynamic-form que utiliza a propriedade p-load, inclusive eu tentei simular aqui porém o comportamento está conforme o esperado, consegue simular pelo StackBlitz (https://stackblitz.com/edit/po-ui)?
